apps/api/src/app/widgets/widgets.controller.ts

```diff
@@ -224,6 +228,27 @@ export class WidgetsController {
     return await this.markMessageAsUsecase.execute(command);
   }
 
+  @ApiOperation({
+    summary: 'Remove a subscriber feed message',
+  })
+  @UseGuards(AuthGuard('subscriberJwt'))
+  @Delete('/messages/remove/:messageId')
```

Comment:
```suggestion
  @Delete('/messages/:messageId')
```

There is no need to add "remove" in rest, delete verb should handle it :)